### PR TITLE
Delegation FAQ

### LINKED ISSUE #489: Write a delegation FAQ
Write a delegation FAQ based on feedback from Mikael.

Can I delegate to more than one baker at a time? 
Can I move from one baker pool to another without having to wait for the next pay day? 
Can I create a pool on my existing baker node (and how)? 
How much does it cost to begin delegating? 
etc...

### PR DESCRIPTION
**Pull request by dg-concordium**
## Purpose

To add a FAQ for delegation.

## Changes

Added file as a snippet that can be added multiple places.

## Checklist

- [X] My code follows the style of this project.
- [X] The code compiles without warnings.
- [X] I have performed a self-review of the changes.
- [ ] I have documented my code, in particular the intent of the
      hard-to-understand areas.
- [ ] (If necessary) I have updated the CHANGELOG.

## CLA acceptance

_Remove if not applicable.

By submitting the contribution I accept the terms and conditions of the
Contributor License Agreement v1.0
- link: https://developers.concordium.com/CLAs/Contributor-License-Agreement-v1.0.pdf

- [X] I accept the above linked CLA.

Closes #489

### CODE REVIEW

**Comment by dg-concordium** [comment]
Mikael has already been through and reviewed the content and given his approval.

File: source/mainnet/net/snippets/delegation-faq.rst [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+
+**What happens when the maximum stake cap is reached for my baker?**
+These two caps are soft caps in the sense that a pool can have more stake than allowed, but only the allowed amounts are taken into account in the consensus algorithm and when distributing rewards. A pool violating the amount limits will gradually receive less rewards per CCD in the pool as the size of the pool continues to increase.
+
+**How much less/more can I expect to earn when opening my pool to delegators?**
+The amount you earn depends on how much stake your pool is baking. The more delegators, the more stake you have and the greater your chances of being selected to bake a block are. 10% of the additional rewards the delegators’ stake in your pool attracts will go to the pool owner. As you are able to have 2x your own stake delegated to your pool, you are effectively able to increase your rewards by 20% if your pool is full.
+
+Therefore, attracting delegators and managing your baker responsibly will maximize rewards for all. In :ref:`Baker management<baker-pool>` you'll find information about how to be a responsible baker.
+
+**Is there a cool-down period for bakers?**
```
Comment (jens-concordium) [inline source/mainnet/net/snippets/delegation-faq.rst:37]: If you are new to the game, would you know what a cool-down is and what triggers it?

File: source/mainnet/net/snippets/delegation-faq.rst [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+Yes, the cool-down period for bakers is three weeks when reducing stake or stopping baking.
+
+**Can I change my restaking preference if I am in a cool-down period?**
+Yes, you can. Note that anything that is restaked during the cool-down period will be unstaked after the stake reduction takes effect.
+
+**How much does it cost to begin baking?**
+There is no specific fee to begin baking, however, you do need to stake a minimum of 14,000 CCD, and you need to have enough CCD left in your disposable balance to cover transaction fees to generate your baker keys (link to definition?).
```
Comment (jens-concordium) [inline source/mainnet/net/snippets/delegation-faq.rst:44]: Technically it doesn't cost a fee to generate the keys, it costs a fee to register them on chain with the baker transaction

Comment (jens-concordium) [reply source/mainnet/net/snippets/delegation-faq.rst:44]: Also, the link is missing

File: source/mainnet/net/snippets/delegation-faq.rst [hunk 3]
```diff
@@ -0,0 +1,98 @@
...
+Yes. You can delegate to multiple bakers but an account can only have one delegation. To delegate to multiple bakers, you need multiple accounts.
+
+**Can I move from one baker pool to another without having to wait for the next pay day?**
+Changes to the pools take effect every 24 hours. So increasing the stake, moving the stake between pools or between passive delegation and a baking pool all take effect at the :ref:`pay day<glossary-pay-day>`.
+
+**How much does it cost to begin delegating?**
+There is no minimum amount required to delegate stake to a pool or enable passive delegation. You just have to have enough CCD left in your disposable balance to cover transaction fees.
+
+**How much can I expect to earn as a delegator?**
+This depends on the baker pool you select and how reliable this baker is. It also depends on how much stake the baker has. It is important to use `CCDScan <https://ccdscan.io>`_ to research bakers before delegating stake.
```
Comment (jens-concordium) [inline source/mainnet/net/snippets/delegation-faq.rst:60]: Is it obvious enough that peoples delegation amount plays a role, or should that be mentioned?

**Review by jens-concordium — COMMENTED** [review]
Some general remarks:

- Is there a reason the FAQs are found only under the desktop wallet? I think I might look for it in the delegation section of Learn.
- I have a gut feeling that most people will just delegate, hence looking for delegator FAQs. Would it make sense to have those questions first on the FAQ list, and then have baker questions below?

**Review by jens-concordium — APPROVED** [review]

**Pull request merged**